manylinux_v2_centos/Dockerfile

```diff
@@ -0,0 +1,95 @@
+ARG BASE_IMAGE=nvidia/cuda:11.8.0-devel-centos7
+FROM ${BASE_IMAGE}
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV PYENV_ROOT="/pyenv" \
+    PATH="/pyenv/bin:/pyenv/shims:$PATH"
+
+RUN yum update -y && yum install -y epel-release wget gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite sqlite-devel xz xz-devel libffi-devel curl git ncurses-devel numactl numactl-devel openssh-clients libcudnn8-devel zip blas-devel lapack-devel protobuf-compiler autoconf automake libtool centos-release-scl scl-utils cmake && yum clean all
+
+# Install tools that have to be installed after the previous install completes.
+RUN yum remove -y git && yum install -y https://packages.endpointdev.com/rhel/7/os/x86_64/endpoint-repo.x86_64.rpm && yum install -y git jq devtoolset-11 && yum remove -y endpoint-repo
+
+# Install latest openssl (can't use the package manager because CentOS 7 stopped after openssl 1.0)
+RUN pushd tmp && \
+    wget https://ftp.openssl.org/source/openssl-1.1.1k.tar.gz && \
+    tar -xzvf openssl-1.1.1k.tar.gz &&\
+    cd openssl-1.1.1k && \
+    ./config --prefix=/usr --openssldir=/etc/ssl --libdir=lib no-shared zlib-dynamic && \
+    make && \
+    make install && \
+    popd
```

Comment:
To keep the final image as small as possible, we should clean up the tarbal and any build artifacts in this step.

If we do that, the `pushd/popd tmp` is probably also unneeded.